app/manifest.json

```diff
@@ -71,7 +71,8 @@
     "activeTab",
     "webRequest",
     "*://*.eth/",
-    "notifications"
+    "notifications",
+    "tabs"
```

Comment:
Requesting new permissions is super hazardous, because it will re-prompt all our users to re-approve permissions. Last time this resulted in a 10% drop off of all our users! Please check if there’s a way to request this permission dynamically at runtime.